src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableQueue.cs

```diff
@@ -34,47 +35,69 @@ public static ImmutableQueue<T> Create<T>()
         [Pure]
         public static ImmutableQueue<T> Create<T>(T item)
         {
-            return ImmutableQueue<T>.Empty.Enqueue(item);
+            return new ImmutableQueue<T>(ImmutableStack.Create(item), ImmutableStack<T>.Empty);
```

Comment:
@stephentoub This will be inlined while `ImmutableQueue.Enqueue` will not be because it has to handle the case where `this` is not empty. I'll post this in the benchmarks.